javascript/fluent.js

```diff
@@ -42,9 +42,11 @@
 				// Create options
 				$.each(fluentLocales, function(locale, name){
 					var item = $("<li><a></a></li>");
+					var lables = "<span class='full-title'>"+name+"</span><span class='short-title'>"+locale.split("_")[0]+"</span>";
```

Comment:
Ah, a slight XSS risk here; We should not trust `name` to be XML escaped.

Try making labels a html string and setting the properties this way, safely, using `.text()` (which does the escaping for us).


```javascript
var item = $("<li><a><span class='full-title'></span><span class='short-title'></span></a><li>");
$(".full-title", item).text(name);
$(".short-title", item).text(locale.split("_")[0]);
$("a", item)
    .attr('data-locale', locale)
	.attr('title', name);
$(".cms-fluent-selector-locales", selector).append(item);
```